src/cleanlab_codex/client.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Optional
+
+from cleanlab_codex.internal.utils import client_from_api_key
+from cleanlab_codex.project import Project
+
+if TYPE_CHECKING:
+    from cleanlab_codex.types.organization import Organization
+
+
+class Client:
+    def __init__(self, api_key: str, organization_id: Optional[str] = None):
+        """Initialize user-level access to the Codex SDK.
+
+        Args:
+            api_key (str): The API key for authenticating the user. (TODO: link to docs on what this means)
+            organization_id (str): The ID of the organization to create projects in. If not provided, the user's default organization will be used.
```

Comment:
In the future, this client may expose more functionality than just creating projects that uses the organization ID. Can we update this to be more general? (doesn't need to be this exact wording)

```suggestion
            organization_id (str): The ID of the organization the client should use. If not provided, the user's default organization will be used.
```